internal/dataplane/parser/translate_secrets.go

```diff
@@ -0,0 +1,102 @@
+package parser
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"time"
+
+	"github.com/kong/go-kong/kong"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/kong/kubernetes-ingress-controller/v2/internal/dataplane/kongstate"
+	"github.com/kong/kubernetes-ingress-controller/v2/internal/store"
+)
+
+func getCACerts(log logrus.FieldLogger, storer store.Storer, plugins []kongstate.Plugin) []kong.CACertificate {
```

Comment:
Do you mind adding a comment to this func?

Reply:
Added: [83ce558](https://github.com/Kong/kubernetes-ingress-controller/pull/3063/commits/83ce558a74cc6b0b30957054fecf84e99a3ad48e)